src/components/verify-tab/VerifyTabPage.vue

```diff
@@ -5,70 +5,83 @@
     @dragleave.prevent="fileDragLeave"
     @drop.prevent="fileDropped"
   >
-    <div class="banner">
-      <div class="text">
-        This page lets you verify whether the OTA package you downloaded is signed with a LineageOS
-        private key and hasn't been tampered with.<br />
-        NOTE: This only works for OTA packages; it won't work with boot.img, recovery.img, etc.
+    <div class="list-container" data-simplebar>
+      <div class="list">
+        <div class="header">
+          <h1>Verify builds</h1>
+          <div>
+            <p>
+              Verify downloaded builds are signed with LineageOS private keys and are untampered by
```

Comment:
not sure if I like the way this sentence starts.

Reply:
"You can verify that downloaded builds..."?

Reply:
maybe!